ports/nrf/mpconfigport.h

```diff
@@ -126,27 +126,27 @@
 
 #define MICROPY_KBD_EXCEPTION                    (1)
 
-#ifndef MICROPY_PY_HW_RNG
-#define MICROPY_PY_HW_RNG                        (1)
-#endif
-
 #define MICROPY_ENABLE_EMERGENCY_EXCEPTION_BUF   (1)
 #define MICROPY_EMERGENCY_EXCEPTION_BUF_SIZE     (0)
 
 // Scan gamepad every 32ms
 #define CIRCUITPY_GAMEPAD_TICKS 0x1f
 
-// if sdk is in use, import configuration
 #if BLUETOOTH_SD
-#include "bluetooth_conf.h"
+#define MICROPY_PY_BLEIO                         (1)
+#define MICROPY_PY_BLE_NUS                       (0)
+#define MICROPY_PY_UBLUEPY                       (1)
+#define MICROPY_PY_UBLUEPY_PERIPHERAL            (1)
+#define MICROPY_PY_UBLUEPY_CENTRAL               (1)
+#define BLUETOOTH_WEBBLUETOOTH_REPL              (0)
 #endif
 
-#ifndef MICROPY_PY_UBLUEPY
-#define MICROPY_PY_UBLUEPY                       (0)
+#ifndef MICROPY_PY_BLEIO
+#define MICROPY_PY_BLEIO                        (0)
 #endif
 
-#ifndef MICROPY_PY_BLE_NUS
-#define MICROPY_PY_BLE_NUS                       (0)
+#ifndef MICROPY_PY_UBLUEPY
+#define MICROPY_PY_UBLUEPY                      (0)
```

Comment:
Q: are we going to get rid of `ubluepy` at some point or does it serve some purpose in other nrf ports?

Reply:
Yes, next step is port `ubluepy` classes to `bleio`, I just like to keep things separated and do them one step at a time.

Reply:
OK, sounds good!